app/Models/Litter/Categories/Political.php

```diff
@@ -2,9 +2,12 @@
 
 namespace App\Models\Litter\Categories;
 
-use Illuminate\Database\Eloquent\Model;
+use App\Models\Litter\LitterCategory;
 
-class Political extends Model
+class Political extends LitterCategory
```

Comment:
This category can be deleted

Reply:
While we don't use these tags anymore, there might be existing photos tagged with Drugs or Political tags. If we're going to delete a category model, shouldn't we delete the DB columns on photos too, or even the drugs and political table?
I think we can push this deletion for a little later. Staging is already way ahead of master.

Reply:
Good point. I am 99% sure they are not used. I will take a look, not important for now